apps/meteor/client/sidebar/footer/voip/VoipFooter.tsx

```diff
@@ -74,6 +65,13 @@ export const VoipFooter = ({
 		togglePause(!paused);
 	};
 
+	const holdTitle = useMemo(() => {
+		if (!isEnterprise) {
+			return t('Hold_EE_only');
+		}
+		return paused ? t('Resume') : t('Hold');
+	}, [isEnterprise, paused, t]);
+
```

Comment:
do we really need this memo here?

Reply:
@ggazzo Not necessarily, I'm using it mostly for the function. I can remove the`useMemo`, but I'll keep the fn, seems more organized that way.